diff --git a/lib/core.go b/lib/core.go
index a43d3c8..a218f43 100644
--- a/lib/core.go
+++ b/lib/core.go
@@ -3,3 +3,9 @@ package lib
 func Base() int {
 	return 1
 }
+
+func Extra() int {
+	return Base() + 1
+}
+
+type Config struct{}
diff --git a/lib/use.go b/lib/use.go
new file mode 100644
index 0000000..7295228
--- /dev/null
+++ b/lib/use.go
@@ -0,0 +1,5 @@
+package lib
+
+func UseExtra() int {
+	return Extra()
+}
